src/components/notification/Notification.js

```diff
@@ -103,7 +118,7 @@ class Notification extends React.Component<Props> {
         const classes = classNames('notification', type, overflow);
 
         return (
-            <div className={classes}>
+            <div className={classes} onKeyDown={this.handleKeyDown} role="presentation">
```

Comment:
I'm curious, why did you decide to add a role of presentation to the div?

Reply:
This needed to be assigned a role to be treated as an interactive banner. The closest role I could find for this was "presentation".

Reply:
What about "alertdialog"